src/components/cars/AddNewCar/AddNewCar.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import { createNewCar } from '../../../redux/cars/AddNewCarSlice';
+import './AddNewCar.css';
+
+const AddNewCar = () => {
```

Comment:
- [ ] As a normal user, I am able to successfully add and remove cars:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/95350c79-59b8-4204-9f41-292dc57e1243)

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/c6a51c2d-2cfe-4196-a7bc-9d883ebc2777)

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/ac774c45-6de0-447c-959b-3fdc863ad09a)

According to the project requirements for teams of 5 people, only admin users should be able to add and remove cars.  This is a core [requirement](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements), kindly implement this functionality.

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/176651b4-cd4a-446c-9012-e1fb92da9e35)